hail/python/hail/docs/change_log.md

```diff
@@ -33,6 +33,18 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.118
+
+Released 2023-05-30
+
+### Bug Fixes
+
+- (hail#13113) `hail.ggplot2.geom_point` now displays a legend group for a column even when it has only one value in it.
+- (hail#13089) Requester pays buckets now work in `hailtop.fs` and `hl.hadoop_*`. This has been broken since at least 0.2.115.
+- (hail#12995) Fixed bug causing poor performance and memory leaks for `Matrix.annotate_rows` aggregations.
```

Comment:
What about #12569 as new features? 
#13081 as a new feature?

I don't think #13126 is a bug fix, but @chrisvittal can chime in?

Reply:
#13113 was the reason I wanted to cut a release in the first place, so I'd like to wait for it to go in first.

Reply:
Added changelog messages for https://github.com/hail-is/hail/pull/12569 and https://github.com/hail-is/hail/pull/13126. I'll change the date as well once this is actually ready to go in. This PR is WIP pending the merge of at least https://github.com/hail-is/hail/pull/13113, https://github.com/hail-is/hail/pull/13147, and any other PRs that are necessary to make CI work again. We could also wait for https://github.com/hail-is/hail/pull/13089 and https://github.com/hail-is/hail/pull/12995, but I don't have strong opinions either way about that. @danking, thoughts?